.goreleaser.yml

```diff
@@ -6,7 +6,7 @@ builds:
     - linux
   goarch:
     - amd64
-  ldflags: -s -w -X github.com/heptio/sonobuoy/pkg/buildinfo.Version={{.Version}}
+  ldflags: -s -w -X github.com/heptio/sonobuoy/pkg/buildinfo.Version=v{{.Version}}
```

Comment:
It's weird to me that you need to add back the `v` it's definitely in the tag.?.?.? Is there some option in Travis?